src/nspawn/nspawn.c

```diff
@@ -3397,10 +3397,9 @@ static int inner_child(
                 if (r < 0)
                         return log_error_errno(r, "personality() failed: %m");
 #endif
-        } else if (arg_architecture >= 0 && arg_architecture != native_architecture())
-                return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP),
-                                       "Selected architecture '%s' not supported locally, refusing.",
-                                       architecture_to_string(arg_architecture));
+        } else if (!arg_quiet && arg_architecture >= 0 && arg_architecture != native_architecture())
+                log_notice("Selected architecture '%s' not supported locally, running through qemu-binfmt.",
```

Comment:
well, that sounds if we actually knew anything about qemu-binfmt, or even set it up ourselves, but we really don't.

qemu itself calls the concept "qemu userspace emulator", btw. maybe hence say:

`…not supported natively on the local CPU, assuming invocation with qemu userspace emulator in effect.`

or so?